core/schema/directory.go

```diff
@@ -96,6 +96,12 @@ func (s *directorySchema) Install() {
 			Args(
 				dagql.Arg("paths").Doc(`Paths of the files to remove (e.g., ["/file.txt"]).`),
 			),
+		dagql.NodeFunc("exists", s.exists).
```

Comment:
todo: look into changing this to:
```suggestion
		dagql.Node("exists", s.exists).
```